<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridExportDialog</class>
 <widget class="QDialog" name="GridExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ProjectViewer/icons.qrc">
    <normaloff>:/icons/images/logo_fs_64x64.png</normaloff>:/icons/images/logo_fs_64x64.png</iconset>
  </property>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>227</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>File:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leFile"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>39</y>
     <width>180</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="cbNull">
      <property name="text">
       <string>Output NULL values</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Null value:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leNull">
      <property name="text">
       <string>-999.99</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../ProjectViewer/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbNull</sender>
   <signal>toggled(bool)</signal>
   <receiver>leNull</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>GridExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
